optuna/integration/pytorch_lightning.py

```diff
@@ -45,6 +47,11 @@ def on_validation_end(self, trainer: Trainer, pl_module: LightningModule) -> Non
         epoch = pl_module.current_epoch
         current_score = logs.get(self.monitor)
         if current_score is None:
+            message = (
+                "The metric '{}' is not in the evaluation logs for pruning. "
+                "Please make sure you set the correct metric name.".format(self._monitor)
```

Comment:
Is the underscore redundant?
```suggestion
                "Please make sure you set the correct metric name.".format(self.monitor)
```